### ISSUE TITLE
NTR: Pristanic acid high in blood after meal with branched-chain fatty acid (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8269]
New term request Pristanic acid high in blood after meal with branched-chain fatty acid (elthat_101118022005):

**Comment by pnrobinson** [comment]
@MickeySegal
this sounds like the kind of observation that only can be made after the diagnosis is known, and it is so bundled that it is only good for one disease. Not convinced it is that useful?

**Comment by pnrobinson** [comment]
@MickeySegal 
The entry states this:
Branched-chain fatty acid accumulation depends on dietary intake of phytanic acid, which is minimal in formula- & breast-fed infants. Thus, phytanic & pristanic acid levels are normal in a neonate w/ZSD.

So I am wondering if we should have a term such as this

Elevated circulating pristanic acid concentration
Def: Concentration of pristanic acid in the blood circulation above the upper limit of normal.
Comment: This finding can be observed in Zellweger syndrome. Note that branched-chain fatty acid accumulation depends on dietary intake of phytanic acid, which is minimal in formula- and breast-fed infants. Thus, phytanic and pristanic acid levels are normal in a neonate with Zellweger spectrum disorder.

**Comment by MickeySegal** [comment]
That is why we put the caveat about the content of the meal in our term name.  Just using "Elevated circulating pristanic acid concentration" would run the risk of people not understanding the necessary conditions.

**Comment by pnrobinson** [comment]
OK, that's logical, please reopen if the following is not OK

Adding new term: 
- ID: HP:6001095
- label: Elevated circulating pristanic acid concentration following branched-chain fatty acid ingestion
- parents: [Abnormal circulating long-chain fatty-acid concentration (HP:0010964)](https://hpo.jax.org/app/browse/term/HP:0010964)
- pmid: PMID:20301621
- def: Concentration of pristanic acid in the blood circulation above the upper limit of normal. This finding can be observed in Zellweger syndrome. Note that branched-chain fatty acid accumulation depends on dietary intake of phytanic acid, which is minimal in formula- and breast-fed infants. Thus, phytanic and pristanic acid levels are normal in a neonate with Zellweger spectrum disorder.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8269
- synonyms:
